ui/app/models/pki/action.js

```diff
@@ -61,13 +61,13 @@ export default class PkiActionModel extends Model {
   @attr importedIssuers;
   @attr importedKeys;
   @attr mapping;
-  @attr('string', { readOnly: true, masked: true }) certificate;
+  @attr('string', { readOnly: true, displayCard: true }) certificate;
```

Comment:
Changed this to `displayCard` since we're no longer using `<MaskedInput>` so `masked` doesn't make sense. If anyone thinks `displayCard` is a confusing name, happy to hear other name suggestions! Was pondering this for a while

Reply:
I wonder if `certificateCard` or `certificate` or maybe `isCertificate` might be more clear since the `certificate-card` component is being used for these attributes? I think either way what you have is fine since the condition is only handled within pki and not a global component like `FormField`.

Reply:
Thanks for the suggestions! Changing it to `isCertificate`